components/analytics/doughnut_chart.jsx

```diff
@@ -57,7 +57,8 @@ export default class DoughnutChart extends React.PureComponent {
         }
         var el = ReactDOM.findDOMNode(this.refs.canvas);
         var ctx = el.getContext('2d');
-        this.chart = new Chart(ctx, {type: 'doughnut', data: this.props.data, options: {}}); //eslint-disable-line new-cap
+        const dataCopy = JSON.parse(JSON.stringify(this.props.data));
+        this.chart = new Chart(ctx, {type: 'doughnut', data: dataCopy, options: {}}); //eslint-disable-line new-cap
```

Comment:
Small nit, is there any reason why we need `eslint-disable-line new-cap`? Looks to me like we actually follow that convention here: https://eslint.org/docs/rules/new-cap

Reply:
I think you are right, I didn't realize that was there actually, because I was too much focused on my change than in the code surounding. I'll change this anyway because is a good timing for that.

Reply:
Fixed